.github/workflows/deploy.yml

```diff
@@ -70,8 +70,31 @@ jobs:
           docker pull growthbook/growthbook:latest
           docker create --name temp-container growthbook/growthbook:latest
           docker cp temp-container:/usr/local/src/app/packages/front-end/.next/static ./static
+          # This is for the sentry source maps task below.
+          docker cp temp-container:/usr/local/src/app/packages/back-end/dist ./back-end-dist
           docker rm temp-container
           aws s3 sync ./static s3://growthbook-cloud-static-files/_next/static
 
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Generate sentry sourcemaps
+        run: |
+          npm install -g @sentry/cli
+          npm exec sentry-cli -- sourcemaps inject ./back-end-dist
+
+      - name: Create Sentry release
+        uses: getsentry/action-release@v1
+        with:
+          environment: 'production'
+          sourcemaps: './back-end-dist'
+          version: ${{ steps.metadata.outputs.sha }}
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
```

Comment:
I think @jdorn will need to add this one to our secrets. Here is where the token can be created https://growthbook.sentry.io/settings/auth-tokens/